test/parallel/test-https-agent-sockets-leak.js

```diff
@@ -9,9 +10,9 @@ const https = require('https');
 const assert = require('assert');
 
 const options = {
-  key: fs.readFileSync(`${common.fixturesDir}/keys/agent1-key.pem`),
-  cert: fs.readFileSync(`${common.fixturesDir}/keys/agent1-cert.pem`),
-  ca: fs.readFileSync(`${common.fixturesDir}/keys/ca1-cert.pem`)
+  key: fs.readFileSync(fixtures.path('keys', 'agent1-key.pem')),
+  cert: fs.readFileSync(fixtures.path('keys', 'agent1-cert.pem')),
+  ca: fs.readFileSync(fixtures.path('keys', 'ca1-cert.pem'))
```

Comment:
This can be:

```js
key: fixtures.readKey('agent1-key.pem'),
cert: fixtures.readKey('agent1-cert.pem'),
ca: fixtures.readKey('ca1-cert.pem')
```